ci/jscrambler-code-integrity.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This is a basic workflow to help you get started with Using Jscrambler Code Integrity Action,
+# documentation can be found here : https://docs.jscrambler.com/latest/code-integrity/documentation/github-ci-integration
+
+name: Jscrambler Code Integrity
+
+on:
+  release:
+    types: [created]
+
```

Comment:
Can we have workflow level permissions?

Reply:
Sure

Reply:
Pha why you close the issue never finished  never asking you to close whit consent